doc/sedcli.8

```diff
@@ -58,9 +58,17 @@ Activates Locking SP.
 .IP "\fB\-R -d <device> [-i]\fR or \fB\-\-revert --device <device> [--psid]\fR"
 Reverts TPer to Manufactured-Inactivate state using either SID or PSID authority.
 
-.IP "\fB\-L -d <device> -t <locktype>\fR or \fB\-\-lock-unlock --device <device> --locktype {RW|RO|WO|UNLOCK}\fR"
+.IP "\fB\-L -d <device> -t <accesstype>\fR or \fB\-\-lock-unlock --device <device> --accesstype {RO|RW|LK}\fR"
 Changes the lock state for the device. There is a possibility to setup Read lock,
```

Comment:
Please get rid of following sentences:
"Changes the lock state for the device. There is a possibility to setup Read lock, Write lock and Read-Write lock and unlock the device."